test/set-location.spec.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+const Behaviour = require('../apps/fbis-ccf/behaviours/set-location');
+
+describe('Set location behaviour', () => {
+
+  let req;
+  let res;
+  let SetLocation;
+  let testInstance;
+
+  beforeEach(() => {
+    req = {
+      sessionModel: {
+        set: sinon.stub().returns(undefined),
+        get: sinon.stub().returns(undefined)
+      },
+      query: {
+        'in-UK': undefined
+      }
+    };
+
+    res = {};
+  });
+
+  describe('getValues', () => {
+
+    class Base {
+      getValues() {}
+    }
+
+    beforeEach(() => {
+      SetLocation = Behaviour(Base);
+      testInstance = new SetLocation();
+    });
+
+    it('should set `in-UK` flag to false if `in-UK` query is false', () => {
```

Comment:
Imho I would write tests like real use cases and as less technical as possible e.g. (it should set a 'in-UK' flag to false when the query in the url does not have `in-UK` e.g. `fbis.gov.uk/?outside-UK`)